src/Sylius/Behat/Context/Ui/Admin/ManagingCustomersContext.php

```diff
@@ -642,4 +642,45 @@ public function iShouldNotSeeASingleOrderFromCustomer($orderNumber)
             sprintf('Cannot find order with number "%s" in the list.', $orderNumber)
         );
     }
+
+    /**
+     * @Given I do not specify any information
+     */
+    public function iDoNotSpecifyAnyInformation()
+    {
+        // Intentionally left blank.
+    }
+
+    /**
+     * @Then I should not be able to specify it password
+     */
+    public function iShouldNotBeAbleToSpecifyItPassword()
+    {
+        Assert::false(
+            $this->createPage->hasPasswordField(), 
+            'There should not be password field, but it is.'
```

Comment:
The step does say what behaviour is expected and if it's failing, it says that expected behaviour didn't occur. No message is the best message 🌯  🎉